compose/queue/api_data/taskqueue_v1beta2.api

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
This file is a description of the REST API directly from Google with modification for scoping. I will add a better description to the documentation for local development.